schema/queue/configuration/QueueConfiguration.go

```diff
@@ -43,13 +46,111 @@ func (mrc *MessageRouterConfiguration) Init(path string) error {
 		mrc.Logger.Error().Err(err).Msg("Json file reading error for QueueConfig")
 		return err
 	}
-	fail := json.Unmarshal(content, mrc)
+	fail := mrc.UnmarshalJSON(content)
 	if fail != nil {
 		mrc.Logger.Error().Err(err).Msg("Deserialization error for QueueConfig")
 		return err
 	}
 	return nil
 }
+func (mrc *MessageRouterConfiguration) UnmarshalJSON(data []byte) error {
```

Comment:
I think we should change this. We do too much manual work when we can rely on standard serialization.

This should be the main config
```go
type QueueConfig struct {
	RoutingKey string                        `json:"name"`
	QueueName  string                        `json:"queue"`
	Exchange   string                        `json:"exchange"`
	Attributes []string                      `json:"attributes"`
	Filters    []MqRouterFilterConfiguration `json:"filters"`
}
```

This should be the MqRouterFilterConfiguration
```go
type FilterSpec struct {
      Filters map[string]FilterFieldsConfig
}
type MqRouterFilterConfiguration struct {
	Metadata FilterSpec `json:"metadata"`
	Message  FilterSpec `json:"message"`
}
```

And the `UnmarshalJSON` should be implemented for `FilterSpec` type